### PR TITLE
Add support for lvm, cryptsetup, tpm2_tools

### PR DESCRIPTION
**Pull request by Rozzii**
This commit ads elements to support the installation of:
- tpm2_tools
- cryptsetup
- lvm2

### CODE REVIEW

File: jenkins/scripts/dynamic_worker_workflow/ipa_builder_elements/ipa-luks-tpm-support/install.d/99-install-luks-tpm [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -eu
+if [ "${DIB_DEBUG_TRACE:-1}" -gt 0 ]; then
```
Comment (tuminoid) [inline jenkins/scripts/dynamic_worker_workflow/ipa_builder_elements/ipa-luks-tpm-support/install.d/99-install-luks-tpm:4]:
```suggestion
if [[ "${DIB_DEBUG_TRACE:-1}" -gt 0 ]]; then
```

File: jenkins/scripts/dynamic_worker_workflow/ipa_builder_elements/ipa-lvm-support/install.d/99-install-lvm2 [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -eu
+if [ "${DIB_DEBUG_TRACE:-1}" -gt 0 ]; then
```
Comment (tuminoid) [inline jenkins/scripts/dynamic_worker_workflow/ipa_builder_elements/ipa-lvm-support/install.d/99-install-lvm2:4]:
```suggestion
if [[ "${DIB_DEBUG_TRACE:-1}" -gt 0 ]]; then
```

File: jenkins/scripts/dynamic_worker_workflow/ipa_builder_elements/ipa-luks-tpm-support/install.d/99-install-luks-tpm [hunk 3]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -eu
```
Comment (tuminoid) [inline jenkins/scripts/dynamic_worker_workflow/ipa_builder_elements/ipa-luks-tpm-support/install.d/99-install-luks-tpm:3]: Add empty line after

File: jenkins/scripts/dynamic_worker_workflow/ipa_builder_elements/ipa-lvm-support/install.d/99-install-lvm2 [hunk 4]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -eu
```
Comment (tuminoid) [inline jenkins/scripts/dynamic_worker_workflow/ipa_builder_elements/ipa-lvm-support/install.d/99-install-lvm2:3]: Add empty line after.

**Review by tuminoid — COMMENTED** [review]
nits

**Comment by Rozzii** [comment]
Test is not relevant these are just optional elements to install packages during IPA building, by default these elements are not used.
/override metal3-ubuntu-e2e-integration-test-main

**Review by tuminoid — COMMENTED** [review]
/approve

**Pull request merged**